pkg/shell/topnav.jsx

```diff
@@ -172,7 +172,7 @@ export class TopNav extends React.Component {
 
         docItems.push(<Divider key="separator1" />);
         docItems.push(<DropdownItem key="about" component="button"
-                                    onClick={() => Dialogs.show(<AboutCockpitModal />)}>
+                                    onClick={() => Dialogs.run(AboutCockpitModal, {})}>
```

Comment:
Passing no attributes seems to be common, and `run()` gets along fine with `undefined` (aka. not given at all).

Reply:
This is difficult.  The type of the properties object needs to match the type of the dialog, and this is enforced (and used to help type inference).  I don't think we can reasonably let this be `undefined` without having an override present.  We might get away with having a default value of `{}`, though.  Let me try that.